libraries/cli11/include/cli11/CLI11.hpp

```diff
@@ -9420,4 +9420,134 @@ inline std::string Formatter::make_option_usage(const Option *opt) const {
 
 
 
+
+class LeapFormatter : public Formatter {
```

Comment:
Could this be pulled out into a separate header for easier integration in the future? Seems like it could be done without having to modify CLI11.hpp at all. Just have cleos and leap-util include it.

Reply:
This is autogenerated header during build process of our custom CLI11, which includes other features such as autocomplete/etc, not available in vanilla cli11. It is a separate header in https://github.com/AntelopeIO/CLI11 and is merged into this superheader on build. This header can be moved to leap repo for sure, if you think this would look better, I'll do that (and roll back / close PR in CLI11 with inclusion of this header in build and one more PR there to remove LeapFormatter from repo)

Reply:
Ignore my comment then. I have not been paying that close attention to this. I'll let @linh2931 review since he has been tracking these changes.

Reply:
How does that happen? I cloned AntelopeIO/CLI11@bd54546 and the followed the instructions to build it, i.e I did
```
git checkout bd545461aaca877b07867156037838abc18ad8dc
mkdir build
cd build/
cmake -DCLI11_SINGLE_FILE=ON ..
make -j
```
But then
```
grep Leap include/CLI11.hpp
```
has nothing. (it doesn't generate the same file as CLI11.hpp here in this PR)

Reply:
Oh no, clash of merges out of sync, should be [b19d133](https://github.com/AntelopeIO/CLI11/commit/b19d1337a3c837ec7a0fb157df97ce806c97e42c) one sec, I'll update

Reply:
done

Reply:
:+1: yes now i'm getting something very close to what is in the PR